app/assets/stylesheets/blacklight/_facets.scss

```
@@ -170,32 +156,27 @@
   }
 }
 
-
-
 /* style for pivot facet's nested list */
 
-.pivot-facet {
+.facet-values {
   @extend .list-unstyled;
-  @extend .py-1;
-  @extend .px-4;
 }
 
-.facet-leaf-node {
-  margin-left: 1rem;
-  padding-right: 1rem;
-  margin-top: -1.5rem;
+.pivot-facet {
+  &.show {
+    @extend .d-flex;
```

```suggestion
    display: flex;
```

I'm pretty sure that's all d-flex does, so 🤷‍♂️